R/indexing.R

```diff
@@ -329,14 +328,28 @@ length.igraph <- function(x) {
   vcount(x)
 }
 
+expand.grid.unordered <- function(i, j, loops = FALSE, directed = FALSE) {
+  grid <- expand.grid(i = i, j = j)
+  if (!directed) {
+    grid <- unique(data.frame(
+      i = pmin(grid$i, grid$j),
+      j = pmax(grid$i, grid$j)
+    ))
+  }
+  if (!loops) {
+    grid[grid[, 1] != grid[, 2], ]
+  } else {
+    grid
+  }
```

Comment:
For minimal complexity, perhaps:

```suggestion
  if (!loops) {
    grid <- grid[grid[, 1] != grid[, 2], ]
  }
  
  grid
```